api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -102,6 +101,10 @@ public void setUserProvider(Provider<User> userProvider) {
 
   @Override
   public ResponseEntity<ClusterListResponse> listClusters() {
+    if (userProvider.get().getFreeTierBillingProjectStatus() != BillingProjectStatus.READY) {
```

Comment:
You could just return an empty list?

Reply:
This actually doesn't return a list currently, just a single "defaultCluster" which is required. Despite the misleading name.